nilearn/maskers/multi_nifti_labels_masker.py

```diff
@@ -11,12 +11,16 @@
 
 @fill_doc
 class MultiNiftiLabelsMasker(NiftiLabelsMasker):
-    """Class for masking of Niimg-like objects.
+    """Class for extracting data from multiple Niimg-like objects \
+       using labels of non-overlapping brain regions.
 
     MultiNiftiLabelsMasker is useful when data from non-overlapping volumes
     and from different subjects should be extracted (contrary to
     :class:`nilearn.maskers.NiftiLabelsMasker`).
 
+    For more details on the definitions of labels in Nilearn,
+    see the :ref:`region` section.
```

Comment:
I would be tempted (in another PR) to add `label (brain region)` and `maps (brain region)` (as defined in the user guide) as part of the nilearn glossary to make it easier to link to them.

Though obviously this would have to be disinguished from "statistcal maps", "class labels"...